### ISSUE TITLE
[FR] Add a subpanel for last visisted opened tabs

### ISSUE THREAD

**Issue by devgid** [mbnuqw/sidebery#1490]
## Description
A history sub panel already exists but I often forget with which panels and tabs I visited in order and  would like to retrace my steps. 

The current history sub panel has a sort by last visited order. However this is open and closed tabs, and doesn't allow you to close tabs. I only want to see open tabs and want to close tabs.

**Comment by mbnuqw** [comment]
Duplicate of #475

**Comment by mbnuqw** [comment]
Nope, changed my mind. This issue is more specific and about the sub-panel, while another is about the panel of last accessed tabs.

**Comment by devgid** [comment]
@mbnuqw another option in this subpanel is it can be grouped by date or alternatively show the date .

That way I can close tabs by day to try limit my open tabs. 
It is much easier to remember what I did yesterday than 3 month ago which makes it easier to continue my thought process or delete tabs.

**Comment by the-a-man-006** [comment]
For History:
- Yes, history needs to be sortable by order-of-opening and order-of-closing (and order-of-last-visit; History Bumper does it already kinda; without it, if you opened a tab a month ago, used it the whole month, and only yesterday closed it, it would appear in the history of the last month, not yesterday, making it harder to find relevant things in the History). 
- I've mentioned it in the Vivaldi forums a few times too; they have the best History management (at least on the surface, lol). 
Basically, an alternate parallel history sorted by tab closing time, not tab creation time.
- A Navigational History feature, that stores a command-chain of all tab-open, tab-close, tab-switch actions you perform, such that, at the end of the day, you could "replay" it again and watch what you did the whole day. 
[And its usecases aren't just recreational; many a time one wants to learn how one got to learn of, say, Pulsars, for the first time. One searches the first instance of "Pulsar" in one's history, then "replays" the browser-navigational-history-command-chain for the whole day at 10x speed. No browser offers this yet! All this while thousands of programmers stay allocated trying to get raytracing to work! What a gross misallocation of resources.].
- Regardless, it's a browser feature that Firefox should go implement. Nothing Sidebery related.

For Recent Tabs
- I've come to the realization/conclusion that an alternate panel for "By Recent Access" (https://github.com/mbnuqw/sidebery/issues/475#issuecomment-930692542) would be very disorienting, whether used exclusively or pivoted side-by-side to the tree-style-view. 
- That, the creation-history is already implied in the tree-structure. The parent must have existed before and been the spawner of the child. So tree-sort-by-recent (which @mbnuqw already has implemented in https://github.com/mbnuqw/sidebery/issues/643#issuecomment-1983479724) is the best; grants the best of both worlds; in the tree-structure, creational history is visible (child was "created" after the parent), and within siblings, access history is visible (the higher the sibling, the more recently accessed it must have been), and same with subtrees.
- Such an automatic tab sorting (https://github.com/mbnuqw/sidebery/issues/1532) would be the best use of the limited resources. Just requires adding a `sort-tree-by-(user's-preference)` function call (already implemented above) whenever a `tab-switch` (user clicks on another tab) happens. That's it!
